docs/relationships.md

```diff
@@ -5,35 +5,113 @@ title: Relationships
 
 ## Database Relationships
 
-Graphback supports `one-to-one` and `one-to-many` relationships and provides out of the box support for the schema and resolvers accordingly.
+Graphback supports `one-to-one`, `one-to-many` and `many-to-one` relationships and provides out of the box support for the schema and resolvers accordingly.
 
-### One To One
+### OneToOne
+
+The `1:1` relation can be declared by:
 
-The `1:1` relation can be simply declared by:
 ```graphql
 type Profile {
   user: User!
 }
+
+type User {
+  profile: Profile!
+}
 ```
-which will create the relationship via a column in the `user` table using column `profileId`. You can customize the field which tracks the relationship using directives.
+
+which will create the relationship via a `userId` column in the `profile` table and a `profileId` column in the `user` table. You can customize the field which tracks the relationship using annotations:
+
 ```graphql
 type Profile {
-  user: User! @OneToOne(field: "yourCustomField")
+  profileUser: User!
+}
+
+type User {
+  """
+  @db.oneToOne: 'profileUser'
+  """
+  profile: Profile!
 }
 ```
 
-### One To Many
+This creates a `1:1` relationship between `profile.profileUserId` and `user.profileId`.
+
+### ManyToOne
+
+The `m:1` relation is declared like so:
 
-The `1:m` relation is declared with array syntax.
 ```graphql
-type Note {
-  comment: [Comment!]!
+type Comment {
+  note: Note!
 }
 ```
-This creates a column `noteId` in table `comment` to track the relationship. You can similarily customize the column name as follows
+
+This creates a `noteId` column in the `comment` table to track the relationship.
+
+You can also customise the field name:
+
 ```graphql
+type Comment {
+  customNote: Note
+}
+```
+
+This creates `customNoteId` in the `comment` table.
+
+You can track to custom field names in the relation table using the `@db.manyToOne` annotation.
+
+```gql
+type Comment {
+  """
+  @db.manyToOne: 'noteComments'
```

Comment:
```suggestion
  @db.manyToOne: 'commentNoteId'
```

Reply:
I did not made sense the othever way arround (suggested relationship will be the other way)

Reply:
```
type Comment {
  """
  @db.manyToOne: 'noteComment'
  """
  comments: [Comment]
}

type Note {
  noteComment: [Comment]
}
```

`@db.manyToOne` is used to track the field in the relation table